test/e2e/e2e_suite_test.go

```diff
@@ -24,7 +25,7 @@ var (
 )
 
 const (
-	testSummaryOutputEnvVar = "GITHUB_STEP_SUMMARY"
+	testSummaryOutputEnvVar = "E2E_SUMMARY_OUTPUT"
```

Comment:
Nice 👍 
I think is better keep generic since we can run the same with or not GITHUB
Very cool